File: abc/abc321/a/main.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    string N;
    cin >> N;
    char prev_digit = 'A';

    for (char digit : N) {
        if (digit >= prev_digit) {
            cout << "No" << endl;
            return 0;
        }
        prev_digit = digit;
    }
    cout << "Yes" << endl;
    return 0;
}

File: abc/abc321/a/main.go
package main

import (
	"fmt"
)

func main() {
	var N string
	fmt.Scan(&N)
	prevDigit := 'A'

	for _, digit := range N {
		if digit >= prevDigit {
			fmt.Println("No")
			return
		}
		prevDigit = digit
	}
	fmt.Println("Yes")
}
